src/compiler/Error.elm

```diff
@@ -12,6 +12,7 @@ module Error exposing
     , toString
     )
 
+import AST.Common.Located exposing (Region)
```

Comment:
Is this needed after all?

Reply:
It depends: do you want the displaying of the error on another PR ?

Reply:
It can go in a different PR, this is big enough (and might get bigger depending on what we decide to do about the tests and Unwrapped variants)